.github/workflows/rust_release.yaml

```
@@ -8,12 +8,12 @@ jobs:
     if: github.event_name == 'workflow_dispatch' && startsWith(github.ref, 'refs/tags')
     uses: ./.github/workflows/rust_ci.yaml
 
-  publish-to-crate-io:
+  publish-to-crate-io-sift_rs:
```

if we are changing the name might as well fix it to `crates` here and everywhere else
```suggestion
  publish-to-crates-io-sift_rs:
```

---

https://github.com/sift-stack/sift/pull/195/commits/9eb767d0f88ca7be6c9f1388d3cf3e191e03918a